code/modules/mob/living/simple_mob/subtypes/animal/sif/kururak.dm

```diff
@@ -0,0 +1,406 @@
+/mob/living/simple_mob/animal/sif/kururak/proc/detect_instinct()	// Will return the Kururak within 10 tiles that has the highest instinct.
+	var/mob/living/simple_mob/animal/sif/kururak/A
+
+	var/pack_count = 0
+
+	for(var/mob/living/simple_mob/animal/sif/kururak/K in hearers(10, src))
+		if(K == src)
+			continue
+		if(K.stat != DEAD)
+			pack_count++
+			if(K.instinct > src.instinct)
+				A = K
+
+	if(!A && pack_count)
+		A = src
+
+	return A
+
+/mob/living/simple_mob/animal/sif/kururak/proc/pack_gauge()	// Check incase we have a client.
+	var/mob/living/simple_mob/animal/sif/kururak/highest_instinct = detect_instinct()
+	if(highest_instinct == src)
+		add_modifier(/datum/modifier/ace, 60 SECONDS)
+	else
+		remove_modifiers_of_type(/datum/modifier/ace)
+
+/datum/ai_holder/simple_mob/intentional/kururak
+	hostile = FALSE
+	retaliate = TRUE
+	cooperative = TRUE
+	can_flee = TRUE
+	flee_when_dying = TRUE
+
+/datum/ai_holder/simple_mob/intentional/kururak/handle_special_strategical()
+	follow_distance = rand(initial(follow_distance), initial(follow_distance) + 2)
+	var/mob/living/simple_mob/animal/sif/kururak/K = holder
+
+	if(istype(K))
+		var/mob/living/simple_mob/animal/sif/kururak/highest_instinct = K.detect_instinct()
+		if(highest_instinct == K)
+			K.add_modifier(/datum/modifier/ace, 60 SECONDS)
+		else
+			K.remove_modifiers_of_type(/datum/modifier/ace)
+
+		if(holder.has_modifier_of_type(/datum/modifier/ace))
+			if(leader && istype(leader, /mob/living/simple_mob/animal/sif/kururak))	// Kururaks will not follow another kururak if they're the pack leader.
+				lose_follow()
+
+		else if(highest_instinct)
+			set_follow(highest_instinct)
+
+	if(holder.has_modifier_of_type(/datum/modifier/ace))
+		hostile = TRUE
+	else
+		hostile = initial(hostile)
+
+/datum/ai_holder/simple_mob/intentional/kururak/pre_special_attack(atom/A)
+	holder.a_intent = I_HURT
+	if(isliving(A))
+		var/mob/living/L = A
+		if(iscarbon(L) || issilicon(L))
+			if(holder.Adjacent(L))
+				holder.a_intent = I_GRAB
+
+			if(iscarbon(L))
+				var/mob/living/carbon/C = L
+				if(!C.eyecheck())
+					if(holder.a_intent != I_GRAB)
+						holder.a_intent = I_DISARM
+
+			if(issilicon(L) && holder.a_intent != I_GRAB)
+				holder.a_intent = I_DISARM
+
+	else if(istype(A, /obj/mecha))
+		holder.a_intent = I_GRAB
+
+/datum/ai_holder/simple_mob/intentional/kururak/post_melee_attack()
+	if(holder.has_modifier_of_type(/datum/modifier/ace))
+		request_help()
+
+// Kururak Ace modifier, given to the one with the highest Instinct.
+
+/datum/modifier/ace
+	name = "Ace"
+	desc = "You are universally superior, in terms of physical prowess."
+	on_created_text = "You feel superior."
+	on_expired_text = "You feel your superiority lessen..."
+	stacks = MODIFIER_STACK_EXTEND
+
+	mob_overlay_state = "ace"
+
+	max_health_flat = 25
+	max_health_percent = 1.2
+	disable_duration_percent
+	incoming_damage_percent = 0.7
+	incoming_healing_percent = 1.5
+	outgoing_melee_damage_percent = 1.5
+	evasion = 5
```

Comment:
iirc things like accuracy and evasion are no longer tied to 15% steps and are now just regular numbers, meaning this will make it 5% harder to hit. It seemed low so I'm not sure if thats intentional or if you didn't know about the evasion change.